driver/bpf/probe-builder-entrypoint.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# Copyright (C) 2013-2018 Draios Inc dba Sysdig.
+#
+# This file is part of sysdig .
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# Simple script to build the BPF probe. Assumes that all the dependencies
+# and requirements are already satisfied (as they are in the accompanying
+# docker container)
+#
```

Comment:
Since this is an internal script, can you perhaps consider using `set -euo pipefail` or even `set -exuo pipefail`? Typically we do this for every script that's mean to be consumed internally or via tests, otherwise you get confusing output such as:

```
* Building probe bpf_probe.o
/probe-builder-entrypoint.sh: line 34: cd: /driver/bpf: No such file or directory
Building bpf
make: *** No targets specified and no makefile found.  Stop.
** Done building probe
cp: cannot stat 'probe.o': No such file or directory
Probe is in /home/gianluca/.sysdig/
```